packages/react-workers/src/workers/methods.js

```diff
@@ -31,8 +34,10 @@ export function getTileFeatures({
     uniqueIdProperty,
     tileFormat,
     geoColumName,
-    spatialIndex
+    spatialIndex,
+    resultFormat: ResultFormat.GeoJsonFeature
```

Comment:
This means that we keep full geojson features always in memory.

I've chosen this as simplest approach that doesn't _modify_ workers "architecture" too much.

We could ask for full features "on-demand", when export is executed though ... this would require only keeping last view filters in state (which may be not necessarily worst thing).